ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-master.yaml

```diff
@@ -458,6 +462,54 @@ tests:
     env:
       PROJECT_NAME: machine-config-operator
     workflow: openshift-ci-security
+- as: e2e-aws-ovn-upgrade-ocb
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|OWNERS_ALIASES|LICENSE)$
+  steps:
+    cluster_profile: aws
+    env:
+      FEATURE_SET: TechPreviewNoUpgrade
+      OCP_ARCH: arm64
```

Comment:
I was trying to get `arm64` to work some time ago and I wasn't successful with it. However, your mileage may vary.